C/CUDNN/build_10.2.jl

```diff
@@ -5,4 +5,8 @@ platforms_and_sources = Dict(
     Platform("x86_64", "windows") => [
         ArchiveSource("https://developer.nvidia.com/compute/cudnn/secure/8.3.2/local_installers/10.2/cudnn-windows-x86_64-8.3.2.44_cuda10.2-archive.zip",
                       "3810b7a4313614aedd28260785f0ebaa69204bb38fa61e439f75c75a4ba07b3d")],
+    Platform("aarch64", "linux") => [
+        FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/c/cudnn/libcudnn8_8.2.1.32-1+cuda10.2_arm64.deb", "4c1619640e5411fb53e87828c62ff429daa608c8f02efb96460b43f743d64bb8"),
```

Comment:
Unfortunately v8.3.2.44 is not currently available.

Currently v8.0.0.180 and v8.2.1.32 is available.

Reply:
Oddly, v8.2.4.15 is also available (only version for CUDA 10.2 on aarch64 I can find): https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/arm64/

Reply:
@maleadt I've rolled back all binaries to v8.2.1.32 - not sure what you think of this, but if you agree, could you upload the binaries?

Reply:
@maleadt Could we roll back-forth CUDNN similar to what @giordano did in 9e01e747 and 06bf115c ?

Reply:
What's your use case for this JLL? If you want to use it with CUDA.jl, we'll need to maintain separate entries in our Artifacts.toml anyway (since I typically copy all of Artifacts.toml from CUDNN_jll.jl), so it could as well be v8.2.4.15.

Reply:
I'm uploading both:

```
libcudnn8-dev_8.2.1.32-1+cuda10.2_arm64.deb: adf7873edbde7fe293f672ebc65fcec299642950797d18b1c3a89855bb23904e
libcudnn8-dev_8.2.4.15-1+cuda10.2_arm64.deb: 4816575e8a14baf3fc1619cb01c6dc2afd4f1b39210079d271208088ecc48702
libcudnn8_8.2.1.32-1+cuda10.2_arm64.deb: 4c1619640e5411fb53e87828c62ff429daa608c8f02efb96460b43f743d64bb8
libcudnn8_8.2.4.15-1+cuda10.2_arm64.deb: 3f08dbe14c3becb9fa1ed85cd4a080efb30edb0c4beb2fc1a2e14da3f2eefb79
```

Reply:
Wait, why did I do this? These binaries are publicly available, so can just be downloaded by the script.

Reply:
> What's your use case for this JLL? If you want to use it with CUDA.jl, we'll need to maintain separate entries in our Artifacts.toml anyway (since I typically copy all of Artifacts.toml from CUDNN_jll.jl), so it could as well be v8.2.4.15.

Primary use case for this JLL is just providing the CUDNN libraries and header files, to fulfill dependencies for TensorRT and subsequently ONNXRuntime and Torch.

Reply:
> Wait, why did I do this? These binaries are publicly available, so can just be downloaded by the script.

Right, only needed uploads are for the run-file installers for v8.2.1.32 (or v8.2.4.15).

Reply:
Uploaded:
```
cudnn-10.2-linux-x64-v8.2.1.32.tgz: fd6321ff3bce4ce0cb3342e5bd38c96dcf3b073d44d0808962711c518b6d61e2
cudnn-10.2-windows10-x64-v8.2.1.32.zip: 3e70876bdcf44f856d9c9dbdbde07ceec43005e97d5ffe83338b255c8466151f
cudnn-11.3-linux-aarch64sbsa-v8.2.1.32.tgz: e3a0e570cb8ba01d5d45e6eb1ebe29ff22fd5fb8ad45bfe7a448f4f95065ec1e
cudnn-11.3-linux-ppc64le-v8.2.1.32.tgz: 4ee4f2afeaae34fdb06da8d4942a6802aae94ecc51f307292c45966eecbe5fb9
cudnn-11.3-linux-x64-v8.2.1.32.tgz: 39412acd9ef5dd27954b6b9f5df75bd381c5d7ceb7979af6c743a7f4521f9c77
cudnn-11.3-windows-x64-v8.2.1.32.zip: 5b9bf2dc4670fb1519ef55e13da5123f0b6b39fac5e6138e31388b269808d5f2
```

Reply:
Great, thanks. v8.2.4 is in #4574